packages/app/src/runner/aut-iframe.ts

```diff
@@ -425,40 +476,6 @@ export class AutIframe {
     })
   }
 
-  beforeScreenshot = (config) => {
```

Comment:
@mschile I removed the snapshot logic here and moved it to the [event-manager](https://github.com/cypress-io/cypress/pull/21237/files#diff-1b89546d0f18a111a3b1bf5d80cce3ed4c65ad470ca7dc074f7711ca9eb22466R504). The tests passed but I haven't verified everything works personally and I also still need to verify if everything is g2 from this change in component testing. On the surface, does this move look correct?

Reply:
thread [here](https://github.com/cypress-io/cypress/pull/21237/files#r866913863)